swap_meet/vendor.py

```diff
@@ -1,2 +1,52 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        self.inventory.remove(item)
+        return item
+
+    def get_by_category(self, selected_category):
+        items_in_category = [item for item in self.inventory if item.category == selected_category]
+        return items_in_category
```

Comment:
You can directly return expressions instead of assigning them to a variable:

```suggestion
        return [item for item in self.inventory if item.category == selected_category]
```